examples/hosts/app-integration/external-data/src/mock-customer-service/service.ts

```diff
@@ -8,11 +8,35 @@ import { Server } from "http";
 import cors from "cors";
 import express from "express";
 import fetch from "node-fetch";
-import { isWebUri } from "valid-url";
 
-import { MockWebhook } from "../utilities";
 import { assertValidTaskData, TaskData } from "../model-interface";
 
+
+/**
+ * Submits notifications of changes to Fluid Service.
+ */
+export function notifyFluidService(data: TaskData): void {
+	const tinyliciousServicePort = 7070;
```

Comment:
This won't always necessarily be 7070, it depends on how the service is initialized. But in theory, we also want this to support drivers other than tinylicious.

I would recommend updating `initializeCustomerService` to take in a parameter with the URL to the service endpoint that will receive our updates. This will make unit testing the service changes easier too.

I'm not sure what the best way is to get the service endpoint that is robust to port changes, though. @ChumpChief do you have any insight here?